.travis.yml

```diff
@@ -80,4 +80,4 @@ script:
   # travis doesn't check out the branch itself but for our make to work we
   # need a branch or alike. So check create a local one ...
   - git checkout -b we-need-a-branch-for-below-to-work
-  - make
+  - make checks build test clobber
```

Comment:
why this change?  Is it to test 'clobber'?  If so, `make all clobber` might be better as it guaranteed that it does all of the default builds?  Certainly no reason to change the order here over the standard build?

Reply:
already removed!